test/Microsoft.VisualStudio.Threading.Analyzers.Tests/VSTHRD100AsyncVoidMethodAnalyzerTests.cs

```diff
@@ -42,44 +42,6 @@ class Test {
             await Verify.VerifyAnalyzerAsync(test, expected);
         }
 
-        [Fact]
-        public async Task ReportWarningOnAsyncVoidLambda()
```

Comment:
This analyzer is useful for flagging when a given method should be `Func<.., Task>` instead of `Action` or `Func<...>`

Reply:
See [VSTHRD101](https://github.com/microsoft/vs-threading/blob/main/doc/analyzers/VSTHRD101.md).